main/index.html

```diff
@@ -19,14 +19,15 @@
         <link rel="stylesheet" href="static/css/main.css">
         <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
         <link href="static/css/hover-min.css" rel="stylesheet" media="all">
+        <link rel="stylesheet" type="text/css" href="static/css/styling.css">
```

Comment:
this is a very ambiguous file name . Rather make it more specific to the function and add comments

Reply:
changed to cards.css